src/colors.js

```diff
@@ -0,0 +1,57 @@
+import {color} from 'chart.js/helpers';
+import {getGradientData} from './helpers';
+
+const rgbs = (c) => Math.round(c._rgb.a * 255) << 24 | c._rgb.r << 16 | c._rgb.g << 8 | c._rgb.b;
+// IEC 61966-2-1:1999
+const toRGBs = (l) => l <= 0.0031308 ? l * 12.92 : Math.pow(l, 1.0 / 2.4) * 1.055 - 0.055;
+// IEC 61966-2-1:1999
+const fromRGBs = (srgb) => srgb <= 0.04045 ? srgb / 12.92 : Math.pow((srgb + 0.055) / 1.055, 2.4);
+
+function interpolate(percent, startColor, endColor) {
```

Comment:
Is this equivalent to `color(startColor).mix(endColor, percent)`?

Reply:
> Is this equivalent to `color(startColor).mix(endColor, percent)`?

I didn't check that color function... let me try! it will simplify a lot!

Reply:
I doesn't seem it's working as I'm expecting:

color(startColor).mix(endColor, percent)

![colorMix](https://user-images.githubusercontent.com/11741250/160590782-fb66bd08-8e0b-41b0-9cf9-9abaa86f1a65.png)

function interpolate(percent, startColor, endColor) {

![colorNoMix](https://user-images.githubusercontent.com/11741250/160590842-4433fcc2-3279-4193-9ae2-0e22768e09ea.png)

As you can see, it seems mix is not precise enough.

Reply:
Oh right, its not the same.

There are few things in the color lib that you can use though. like set/get the rgb property of a color.
I think I'll add that `interpolate` to it :)

Reply:
FINE! I have got that code in my lib as well and is based to IEC 61966-2-1:1999 ;)